src/utils/transformStream-polyfill.js

```diff
@@ -0,0 +1,32 @@
+const _jstds_wm = new WeakMap(); /* info holder */
```

Comment:
Is the reason for this obvious to front-end devs? It's definitely not clear to me

Reply:
Not sure how obvious it is, but the purpose is to polyfill the [TextDecoderStream](https://developer.mozilla.org/en-US/docs/Web/API/TextDecoderStream) API in browsers that don't provide it natively. I ran into this issue because the JournalClient from `data-plane-gateway` uses `TextDecoderStream`, which wasn't available in Firefox I guess?

Reply:
I moved to a stand alone folder... but polyfills are fairly common in the front end world.